index.html

```diff
@@ -2106,6 +2116,11 @@ <h3>Processing capabilities</h3>
        used. The intent is that if this is not possible a new session
        will not be established.
 
+      <dt>"<code>userAgent</code>"
+      <dd><p>return <a>success</a> with data <a><code>null</code></a>.
+
+        <p class=note>Matching is not supported for this capability.
+
```

Comment:
Shall we leave this out for now until it has been agreed on how to handle https://github.com/w3c/webdriver/issues/1792?

Reply:
+1 for leaving this out until we sort out the (pre-existing, broader) issue.

Reply:
Removed this part for now.